sentry.server.config.js

```diff
@@ -0,0 +1,21 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  dsn:
+    SENTRY_DSN ||
+    'https://6935d02f879c4536aecfe8fa07ae1d3a@o1270901.ingest.sentry.io/6462188',
+  // Adjust this value in production, or use tracesSampler for greater control
+  tracesSampleRate: 0.1,
```

Comment:
do we only want to sample 10% of users?

Reply:
We can't afford more on our free plan. Even this might be too much.